lib/utils/local-process.js

```diff
@@ -28,6 +29,14 @@ class LocalProcess extends ProcessManager {
      * @public
      */
     start(cwd, environment) {
+        const stat = fs.lstatSync(path.join(cwd, 'content'));
+
+        // Check that content folder is owned by the current user
+        if (stat.uid !== process.getuid()) {
```

Comment:
Isn't `process.getuid()` not a thing for Windows 🤔

Reply:
ah yup - had to fix that in another place earlier today. Good 🎣 !